src/log_migration.rb

```diff
@@ -44,3 +45,12 @@
         end
     end
 end
+time = Time.utc 2015,8,26
+LogTools::Converter.register "converter for DepthMap missing the time field", time, Orocos.registry do
+    conversion "/base/samples/DepthMap","/base/samples/DepthMap" do |dst,src|
+        deep_cast(dst,src,:self)
+        if not src.timestamps.empty? then
```

Comment:
The date will be ... fuzzy (e.g. some people may generate logs in the next weeks that don't have the time field)

You should check that the time field exists with (needs testing !) `dst.class.has_field?('time')`